### DISCUSSION TITLE
Tor access in case of DNS

### DISCUSSION THREAD

**Discussion by HowlingMadZ** [upptime/upptime#208] [Ideas] [answered]
In case you lose DNS rights, it might be interesting to have a v2 and v3 .onion address for both the status, and other services. I know proton mail does this for their main site, so people can access it from difficult locations. But it's also good for difficult situations like DNS removals.

(Sorry if this is the wrong place to write this, it was the only place I could find to make suggestions)

-- edit --

Sorry, I thought I was on a specific website, not on the library repo.

**Comment by AnandChowdhary** [comment, marked as answer]
Ok, closing this. 😄
